### DISCUSSION TITLE
Can the distance of the rows and the grid be changed?

### DISCUSSION THREAD

**Discussion by AaronVera1** [xournalpp/xournalpp#2710] [Q&A] [answered]
I am starting to use xournal ++, it is excellent software, I find myself needing to change the distance between the lines and the size of the grid. Is there a way to do it?
Thank you.

**Comment by rolandlo** [comment, marked as answer]
The size of the grid for the graph paper can be changed. You have to define a new page template in the `pagetemplate.ini` file, see #2137. You can also make the grid size for snapping accordingly by adjusting the grid size in the drawing area panel in the preferences.

For the ruled paper background the distance between the lines cannot be changed currently.
